.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Description of what this PR fixes, steps to reproduce.
+
+```
```

Comment:
Let's wrap hints, comments, and helpers like this code block in comments (`<!--- hint/helper -->`) so that we don't need to delete them every time we're opening a PR.